monkey/monkey_island/cc/ui/src/components/utils/SafeOptionValidator.js

```diff
@@ -19,6 +14,11 @@ function getPluginDescriptors(schema, config) {
       name: 'Exploiters',
       allPlugins: schema.properties.propagation.properties.exploitation.properties.exploiters.properties,
       selectedPlugins: Object.keys(config.propagation.exploitation.exploiters)
+    },
+    {
+      name: 'CredentialsCollectors',
+      allPlugins: schema.properties.credentials_collectors.properties,
+      selectedPlugins: Object.keys(config.credentials_collectors)
```

Comment:
If we change the configuration path of something, we'll need to update to here as well as `PluginSelectorTemplate.tsx`.

I see that the way we access the configuration is different in both places, but we should have the paths at one place and create a utility method that accepts the path as a parameter which we can use in this file. Maybe in a different PR.